ui/qml/pages/HeartratePage.qml

```diff
@@ -85,45 +85,65 @@ PagePL {
 
         //Type summary
         Grid {
-            columns: 2
+            columns: 3
             spacing: styler.themePaddingMedium
-            width: parent.width
-            LabelPL { text: qsTr("Relaxed") }
-            Item { width: parent.width * 0.5; height: 50
+            width: parent.width - (styler.themePaddingMedium * 2)
+            LabelPL {text: qsTr("Relaxed")}
+            Item { 
+                width: parent.width * 0.5
+                height: 50
                 Rectangle { color: "grey"; width: parent.width * (relaxed  / total) ; height: parent.height }
-                LabelPL { text: Math.floor((relaxed / total) * 100) + "%"; anchors.centerIn: parent}
+                LabelPL { text: Math.round((relaxed / total) * 100) + "%"; anchors.centerIn: parent}
             }
-            LabelPL { text: qsTr("Light") }
-            Item { width: parent.width * 0.5; height: 50
+            LabelPL {text: qsTr("<= %1 BPM".arg(Math.round(maxHR()*0.5)))}
```

Comment:
I would prefer use of ≤ symbol.
```
-            LabelPL {text: qsTr("<= %1 BPM".arg(Math.round(maxHR()*0.5)))}
+            LabelPL {text: qsTr("≤ %1 BPM".arg(Math.round(maxHR()*0.5)))}
```

Reply:
Thanks for that suggestion. Yes, that looks much better. :+1: 

![grafik](https://github.com/user-attachments/assets/1165e0bf-e136-471c-acd4-9b8251e5fa2c)